x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -833,13 +833,15 @@ func TestBuildMappings(t *testing.T) {
         value:
           field: colliding_field
 `,
-			err:      fmt.Errorf(`collision between label field "colliding_field" and value field "colliding_field"`),
+			err:      errors.New(`collision between label field "colliding_field" and value field "colliding_field"`),
 			expected: nil,
 		},
 	} {
 		t.Run(test.title, func(t *testing.T) {
 			var mappings []StatsdMapping
-			err := yaml.Unmarshal([]byte(test.input), &mappings)
+			if err := yaml.Unmarshal([]byte(test.input), &mappings); err != nil {
+				t.Error(err)
+			}
```

Comment:
Since this test is using assert/require, do you think it would be more idiomatic to keep the test consistent by only using one of those libraries?

```suggestion
			err := yaml.Unmarshal([]byte(test.input), &mappings)
			require.NoError(t, err)
```

Reply:
Makes sense. I'll make the change.